lib/pbench/test/unit/server/auth/test_auth.py

```diff
@@ -494,62 +493,33 @@ def record_abort(code: int, message: str = ""):
             )
             assert record["code"] == expected_code
 
-    def test_verify_auth_exc(self, monkeypatch, make_logger):
-        """Verify exception handling originating from verify_auth_internal"""
-
-        def vai_exc(token_auth: str) -> Optional[User]:
-            raise Exception("Some failure")
-
-        monkeypatch.setattr(Auth, "verify_auth_internal", vai_exc)
-        app = Flask("test-verify-auth-exc")
-        app.logger = make_logger
-        with app.app_context():
-            user = Auth.verify_auth("my-token")
-        assert user is None
-
-    def test_verify_auth_internal(self, make_logger, pbench_drb_token):
-        """Verify success path of verify_auth_internal"""
-        app = Flask("test-verify-auth-internal")
+    def test_verify_auth(self, make_logger, pbench_drb_token):
+        """Verify success path of verify_auth"""
+        app = Flask("test-verify-auth")
         app.logger = make_logger
         with app.app_context():
             current_app.secret_key = jwt_secret
             user = Auth.verify_auth(pbench_drb_token)
         assert str(user.id) == DRB_USER_ID
```

Comment:
Admittedly, you didn't add or change this, but `user.id` is already a string, so we don't need the conversion, here.